internal/util/debug/debug.go

```diff
@@ -37,11 +37,20 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+var (
+	graphsPath  = "/debug/graphs"
```

Comment:
Why do we have those variables and why they are variables?

Reply:
Changed to constants. And used these constants everywhere in the code. Some if the URL are used in test as well. hence converted all the URL to constants and used uniformly across.